cloud-agent/service/server/src/main/scala/org/hyperledger/identus/pollux/credentialschema/SchemaRegistryEndpoints.scala

```diff
@@ -103,19 +129,19 @@ object SchemaRegistryEndpoints {
 
   val updateSchemaEndpoint: Endpoint[
     (ApiKeyCredentials, JwtCredentials),
-    (RequestContext, String, UUID, CredentialSchemaInput),
+    (RequestContext, UUID, CredentialSchemaInput),
     ErrorResponse,
-    CredentialSchemaResponse,
+    CredentialSchemaResponse | CredentialSchemaDidUrlResponse,
```

Comment:
Does it make sense to unpack the DIDURLResponse and use CredentialSchemaResponse everywhere?